pom.xml

```diff
@@ -15,7 +15,7 @@
 
     <properties>
         <assertj.version>2.6.0</assertj.version>
-        <item-renderer.version>3.0.6</item-renderer.version>
+        <item-renderer.version>3.0.7</item-renderer.version>
```

Comment:
Yep - This is the correct TDS_ItemRenderer version, which was addressed by [this PR](https://github.com/SmarterApp/TDS_ItemRenderer/pull/26).  We talked about this offline, so I'm going to merge this into **develop**.